modules/@apostrophecms/doc/index.js

```diff
@@ -221,32 +221,22 @@ module.exports = {
       '@apostrophecms/doc-type:afterDelete': {
         // If deleting draft also delete published, but
         // not vice versa ("undo publish" is a thing).
-        async deleteOtherModeAfterDelete(req, doc) {
+        async deleteOtherModeAfterDelete(req, doc, options) {
```

Comment:
`deleteOtherModeAfterDelete` A bit misleading since it deals with  more than two modes.